### PR TITLE
mainnet runtime updated

### PR DESCRIPTION
**Pull request by ksr30**
Solved #18

### CODE REVIEW

**Review by ayushmishra2005 — APPROVED** [review]

File: runtime/mainnet/src/lib.rs [hunk 1]
```diff
@@ -435,12 +457,14 @@ construct_runtime!(
...
 		Sudo: pallet_sudo::{Module, Call, Config<T>, Storage, Event<T>},
-		GenericAsset: pallet_generic_asset::{Module, Call, Storage, Config<T>, Event<T>},
+		CustomAsset: polkadex_custom_assets::{Module, Call, Storage, Config<T>, Event<T>},
 		Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>, ValidateUnsigned},
 		Session: pallet_session::{Module, Call, Storage, Event, Config<T>},
 		Historical: pallet_session_historical::{Module},
 		// Include the custom logic from the polkadex pallet in the runtime.
 		Polkadex: polkadex::{Module, Call, Storage, Event<T>},
+		PolkadexUniswap: polkadex_swap_engine::{Module, Call, Storage, Event<T>},
+        PolkadexIdentity: pallet_idenity::{Module, Call, Storage, Event<T>},
```
Comment (ayushmishra2005) [inline runtime/mainnet/src/lib.rs:467]: Please correct the formatting here

File: runtime/mainnet/src/lib.rs [hunk 2]
```diff
@@ -647,32 +671,32 @@ impl_runtime_apis! {
 
 	impl runtime_api::DexStorageApi<Block> for Runtime{
 
-	    fn get_ask_level(trading_pair: Hash) -> Result<Vec<FixedU128>,ErrorRpc> {
+	    fn get_ask_level(trading_pair: (Hash,Hash)) -> Result<Vec<FixedU128>,ErrorRpc> {
```
Comment (ayushmishra2005) [inline runtime/mainnet/src/lib.rs:674]: Formatting issue

File: runtime/mainnet/src/lib.rs (hunk 1 above)
Comment (ksr30) [reply runtime/mainnet/src/lib.rs:467]: It's not showing on IDE. 
Screenshot from 2021-01-13 17-04-12

**Comment by ksr30** [comment]
[Please Review the PR for Mainnet upgradation.](https://trello.com/c/Gw4ziIOL/87-please-review-the-pr-for-mainnet-upgradation)

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**